submodules/islandora_webform_embargo/islandora_webform_embargo.module

```diff
@@ -22,7 +22,7 @@ function _get_islandora_webform_embargo_settings($nid, $sid = FALSE) {
     $single_record = $defaults->execute()->fetchObject();
   }
 
-  return $single_record ?? FALSE;
+  isset($single_record) ? return $single_record : return FALSE;
```

Comment:
That line is strange

maybe?
(Brandon can you test by replacing with this line?)
```PHP
return $single_record ? $single_record : FALSE;
````

But i would also initialize  $single_record on top, or even return early if not $nid

Reply:
@DiegoPino That change fixes the Line 25 problem, but we still have a PHP error:

> PHP Parse error:  syntax error, unexpected '?' in islandora_webform_embargo.module on line 91

The line in question:
`'#default_value' => ($defaults && $defaults->embargo_type == 0) ? date_parse($defaults->embargo_date ?? "now") : NULL,`

Reply:
@bondjimbond FYI, me just lurking here (not sure why i get every existing notification of internet) but not a mantainer.

Reply:
@bondjimbond i get it. Will see if anyone else chimes in here, if not, its just a global ?? find and replace with a isset(). etc. That said, any chance your provider can move you to php 7? 5.6 has been deprecated for 18 months or so already and you will be missing a LOT of features and future modules. Even the patches that can come to Drupal 7 before it dies will be probably for PHP 7+

Reply:
@bondjimbond I don't have an active PHP 5.x environment to build and test this with either. If you issue a new PR with all the necessary changes I'd be happy to accept it, though.